lib/msal-browser/src/cache/BrowserCacheManager.ts

```diff
@@ -400,26 +400,60 @@ export class BrowserCacheManager extends CacheManager {
      * Gets the active account
      */
     getActiveAccount(): AccountInfo | null {
-        const activeAccountIdKey = this.generateCacheKey(PersistentCacheKeys.ACTIVE_ACCOUNT);
-        const activeAccountId = this.browserStorage.getItem(activeAccountIdKey);
-        if (!activeAccountId) {
+        const activeAccountKeyNew = this.generateCacheKey(PersistentCacheKeys.ACTIVE_ACCOUNT_NEW);
+        const activeAccountValueNew = this.getItem(activeAccountKeyNew);
+        if (!activeAccountValueNew) {
```

Comment:
A comment would be helpful here to describe that were first looking for an account cached with the old key.